turbinia/config/__init__.py

```diff
@@ -22,16 +22,23 @@
 import os
 import sys
 
+from turbinia import TurbiniaException
+
 DATETIME_FORMAT = '%Y-%m-%dT%H:%M:%S.%fZ'
 
 # Look for config files with these names
-CONFIGFILES = ['.turbiniarc', 'turbinia.conf', 'turbinia_config.py']
-# Look in homedir first, then /etc/turbinia, and finally in the source
-# config dir for config files
+CONFIGFILES = ['.turbiniarc', 'turbinia.conf', 'turbinia_config_tmpl.py']
+# Look in homedir first, then /etc/turbinia
 CONFIGPATH = [
-    os.path.expanduser('~'), '/etc/turbinia',
-    os.path.dirname(os.path.abspath(__file__))
+    os.path.expanduser('~'),
+    '/etc/turbinia',
+    os.path.dirname(os.path.abspath(__file__)),
 ]
+# Config setup reminder for cleaner error handling on empty configs.
+# TODO(johngalvin): Link out to install guide anchors instead?
```

Comment:
Minor nit, but I wouldn't leave a TODO for something small like this since we're not likely to ever come back around and change it.  I could go either way on this since on one hand it's nicer for the user to link out, but on the other hand I don't necessarily want to change code when the docs change.

Reply:
I had it here thinking it might be nice with the upcoming install guide rework, but I'm happy either way -- removed.